tests/unit/_test_categorical_analyze.py

```diff
@@ -1,15 +1,12 @@
-import json
-
 import pandas as pd
-
 from sem_covid.entrypoints.notebooks.EDA.eda_wrangling.categorical_analyze import fast_categorical_analyze
 
 
+# TODO: what are testing here? why plots pop up?
```

Comment:
the function has a display inside which will run once the function is called. We are testing what the function is returning